tardis/conftest.py

```diff
@@ -117,3 +118,30 @@ def included_he_atomic_data(test_data_path):
 def tardis_config_verysimple():
     return yaml_load_config_file(
         'tardis/io/tests/data/tardis_configv1_verysimple.yml')
+
+###
+# HDF Fixtures
+###
+
+@pytest.fixture(scope="session")
+def hdf_file_path(tmpdir_factory):
+    path = tmpdir_factory.mktemp('hdf_buffer').join('test.hdf')
+    return str(path)
+
+@pytest.fixture(scope="session")
+def config_verysimple():
+    filename = 'tardis_configv1_verysimple.yml'
+    path = os.path.abspath(os.path.join('tardis/io/tests/data/', filename))
+    config = Configuration.from_yaml(path)
+    return config
+
+@pytest.fixture(scope="session")
+def simulation_verysimple(config_verysimple, atomic_data_fname):
+    atomic_data = AtomData.from_hdf5(atomic_data_fname)
```

Comment:
I know there is a problem with the atomic data and preparing it multiple times. I think we could solve this by having a session fixture that does `AtomData.from_hdf(...)` and another function scope fixture that takes the first fixture and returns a `copy.deepcopy`. Could that work?